src/__tests__/prove.ts

```diff
@@ -38,9 +40,20 @@ describe('Prove endpoint', () => {
     const message = 'Signature for SealHub'
     const wallet = Wallet.createRandom()
     const signature = await wallet.signMessage(message)
+    const input = await generateInput(signature, message)
+    const data = {} as {
+      TPreComputes: string
+      U: string
+      s: string
+    }
+    Object.entries(input).forEach(([key, value]) => {
+      data[key as keyof typeof data] = JSONbig.stringify(value)
```

Comment:
Let's put the type into separate type or interface

Reply:
Should we? We use it in only one place, here

Reply:
You can import only `stringify` from `json-bigint`:
`import { stringify } from 'json-bigint'`